setup.py

```diff
@@ -42,6 +42,7 @@ def run_tests(self):
       url="http://github.com/confluentinc/ducktape",
       packages=find_packages(),
       package_data={'ducktape': ['templates/report/*']},
-      tests_require=['pytest'],
+      install_requires=['jinja2', 'requests'],
+      tests_require=['pytest', 'jinja2', 'requests'],
```

Comment:
Pretty sure you don't have to specify all the install_requires in the tests_require -- they are combined for test. If you want to verify, you could test this pretty easily in a VM.

It'd also be nice to keep these types of changes isolated to their own changesets, even if they are trivially small.